apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx

```diff
@@ -36,7 +36,16 @@ const PolicyTableRowHeader = ({
   return (
     <div id={table.id.toString()} className="flex w-full items-center justify-between">
       <div className="flex gap-x-4 text-left">
-        <Link href={`/project/${ref}/editor/${table.id}`}>
+        <Link href={`/project/${ref}/editor/${table.id}`} className="flex items-center gap-x-2">
+          {table.rls_enabled ? (
+            <div className="flex items-center gap-x-1 text-xs">
+              <Lock size={14} strokeWidth={2} className="text-brand-500" />
```

Comment:
lets use text-brand here, it looks a bit muted otherwise compared to the unlock icon